Backpack/build.gradle

```diff
@@ -31,6 +31,12 @@ android {
       "-Xuse-experimental=net.skyscanner.backpack.util.ExperimentalBackpackApi",
     ]
   }
+  lintOptions {
+    baseline file("lint-baseline.xml")
+    lintConfig file("../lint.xml")
+    checkReleaseBuilds false
```

Comment:
this would run lint as part of a release build, but as we're already running it separately this isn't needed (it'd just unnecessarily increase build times)

Reply:
Is it possible to put a comment to that effect in the file?